eng/pipelines/templates/jobs/perf.yml

```diff
@@ -31,11 +34,20 @@ extends:
     - template: /eng/pipelines/templates/variables/globals.yml
     Language: Cpp
     LanguageVersion: 'N/A'
-    InstallLanguageSteps: ${{ parameters.InstallLanguageSteps }}
+    InstallLanguageSteps:
+      - pwsh: |
+          Write-Host "##vso[task.setvariable variable=VCPKG_BINARY_SOURCES_SECRET;issecret=true;]clear;x-azblob,https://cppvcpkgcache.blob.core.windows.net/public-vcpkg-container,,read"
+        displayName: Set Vcpkg Variables
+      - ${{ if parameters.Profile }}:
+        - task: Bash@3
+          inputs:
+            targetType: 'inline'
+            script: sudo apt install valgrind -y
```

Comment:
```suggestion
            script: sudo apt install valgrind -y
            displayName: Install valgrind
```

Reply:
Please add a `displayName`, otherwise this just appears as `Bash` in the pipeline step list